woocommerce/woocommerce-android
# [Woo POS] Entry point eligibility conditions improvements

Closes: #11895


### Description

The PR changes the screen size condition for entry to the POS mode. Only devices with:
* Smaller size >= 674
AND
* Bigger size >= 800

are eligible

pfoUAQ-Oq-p2




### Steps to reproduce


### Testing information

* Try different sizes of a screen
* Notice when POS is shown and when not

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/c1057fba-f1cc-46c7-a410-51c247fcc53e
